Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -2257,15 +2257,20 @@ static void webkit_web_view_class_init(WebKitWebViewClass* webViewClass)
      *
      * Emitted after #WebKitWebView::context-menu signal, if the context menu is shown,
      * to notify that the context menu is dismissed.
+     *
+     * Deprecated: 2.48, WPE WebKit does not emit this signal.
      */
-    signals[CONTEXT_MENU_DISMISSED] =
-        g_signal_new("context-menu-dismissed",
-                     G_TYPE_FROM_CLASS(webViewClass),
-                     G_SIGNAL_RUN_LAST,
-                     G_STRUCT_OFFSET(WebKitWebViewClass, context_menu_dismissed),
-                     0, 0,
-                     g_cclosure_marshal_VOID__VOID,
-                     G_TYPE_NONE, 0);
+    signals[CONTEXT_MENU_DISMISSED] = g_signal_new("context-menu-dismissed",
+        G_TYPE_FROM_CLASS(webViewClass),
+#if PLATFORM(GTK)
+        G_SIGNAL_RUN_LAST,
+#else
+        static_cast<GSignalFlags>(G_SIGNAL_RUN_LAST | G_SIGNAL_DEPRECATED),
```

Comment:
Formatter complained about the indentation of this new line despite (to my eyes) matching the existing indentation. So I've decided to just reformat the entire block to clangd's suggested format, hope that's good.

Reply:
Yes, that was the right choice.

Reply:
(Older class init functions often don't comply with our mandatory code style guidelines. We just reformat it when we touch the old code.)